styx-scheduler-service/src/main/java/com/spotify/styx/flyte/FlyteAdminClientRunner.java

```diff
@@ -168,9 +169,11 @@ public String createExecution(final RunState runState, final String execName, fi
         .put(STYX_WORKFLOW_INSTANCE_ANNOTATION, runState.workflowInstance().toKey())
         .put(STYX_EXECUTION_ID_ANNOTATION, styxVariables.get(STYX_EXECUTION_ID))
         .build();
+    // case shouldnt matter because the case is inhereted from the FlyteLaunchPlan
     final var extraDefaultInputs = ImmutableMap.<String, String>builder()
-        .putAll(styxVariables)
-        .putAll(triggeredParams)
+        .putAll(keysToUpperCase(flyteExecConf.inputFields())) // First use the fields stored in the flyteExecConf
```

Comment:
Please create a test that test this code, meaning that if I reorder these three lines the test fails